src/components/Issue.js

```diff
@@ -25,6 +25,14 @@ class Issue extends Component {
 
     return ""
   }
+
+  displayPeopleWithInterest = (issue) => issue.peopleWithInterest && issue.peopleWithInterest.length > 0 ?
+    <span>
+      {" | interested by "}
+      <span className="person">{issue.peopleWithInterest.join(', ')}</span>
```

Comment:
i think it's better if each person name is wrapped by `span.person`. can you change to that? and tenary operator `? : ` isn't suitable because this block is long enough.

also `interested by` is not grammatically correct for this use case, can you change to just `interested: ` instead?

Reply:
Sure. I will update the code when I have time